catalog/data_metastores.go

```diff
@@ -0,0 +1,21 @@
+package catalog
+
+import (
+	"context"
+
+	"github.com/databricks/databricks-sdk-go"
+	"github.com/databricks/terraform-provider-databricks/common"
+	"github.com/hashicorp/terraform-plugin-sdk/v2/helper/schema"
+)
+
+func DataSourceMetastores() *schema.Resource {
+	return common.WorkspaceData(func(ctx context.Context, data *MetastoresData, w *databricks.WorkspaceClient) error {
```

Comment:
So @alexott here I'm switching to `*databricks.AccountClient` but I'm having a weird issue now. Can I use `common.WorkspaceData` with the `AccountClient` context?

Reply:
Is this outdated? What was the context / discussion behind this